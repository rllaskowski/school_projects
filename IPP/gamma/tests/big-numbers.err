ERROR 1
ERROR 3
ERROR 4
