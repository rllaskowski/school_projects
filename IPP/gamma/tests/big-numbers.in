B 4294967295 4294967295 4294967295 4294967295
B 25 25 1000 4294967295
m 1000 0 4294967297
m 1000 4294967298 0
m 1000 0 0
