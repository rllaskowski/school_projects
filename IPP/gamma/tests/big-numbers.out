OK 2
1
